index.html

```diff
@@ -54,34 +55,32 @@
         doorTwoChart = new google.visualization.Gauge(document.getElementById("doorTwoChart"));
         noiseChart = new google.visualization.Gauge(document.getElementById("noiseChart"));
         socket.on('onlineUserCount', function (msg) {
+
           $('#onlineCounter').html(msg);
           onlineUserCountCachedData = msg;
           let chartData = [["Label", "Value"], ["", msg]];
           drawGaugeChart("usersOnlineChart", chartData);
         });
         socket.on('doorOne', function (doorOneAverage, doorOnePassedThroughToday) {
-          let doorOneAvgInteger = Math.floor(doorOneAverage);
-          doorOneCachedAverage = doorOneAvgInteger;
+          if (doorOneAverage > max) {
+            max = doorOneAverage;
+          }
+          doorOneCachedAverage = doorOneAverage;
           $('#doorOne').html(doorOnePassedThroughToday);
-          let chartData = [["Label", "Value"], ["Door one", doorOneAvgInteger]];
+          let chartData = [["Label", "Value"], ["Door one", doorOneAverage]];
           drawGaugeChart("doorOneChart", chartData);
         });
         socket.on('doorTwo', function (doorTwoAverage, doorTwoPassedThroughToday) {
-          let doorTwoAvgInteger = Math.floor(doorTwoAverage);
-          doorTwoCachedAverage = doorTwoAvgInteger;
+          doorTwoCachedAverage = doorTwoAverage;
           $('#doorTwo').html(doorTwoPassedThroughToday);
-          let chartData = [["Label", "Value"], ["Door two", doorTwoAvgInteger]];
+          let chartData = [["Label", "Value"], ["Door two", doorTwoAverage]];
           drawGaugeChart("doorTwoChart", chartData);
         });
         socket.on("noiseData", function (msg) {
           noiseCachedData = msg;
           let chartData = [["Label", "Value"], ["dB", msg]];
           drawGaugeChart("noiseChart", chartData);
         })
-
-        $.get("/getDoorAverageMinutes")
```

Comment:
Here we take the average minutes which are set in the config and show them to the user in the gauge div heading.